sniffles/ruletrafficgenerator.py

```diff
@@ -1586,11 +1586,15 @@ def generate_full_eval(self, rule=None):
                     path = []
                     self.follow_all_branches(nfa, nfa.start, path)
 
-    def generate_nfa_data(self, rule=None):
+    def generate_nfa_data(self, rule=None, length=0):
         if rule:
             data = []
             http_content = []
             content_options = rule.getContent()
+            if content_options is None:
+                if length == -1:
```

Comment:
I think `None` represents "no restriction on length" better than `-1`.

Reply:
The system is currently built around the idea of -1 implying random.  In fact, I believe it is even possible to designate a range using negatives (-10 is a range between 1 and 10).  I cannot remember for sure (that was present in at least on iteration).  However, changing from -1 has potentially far reaching changes.  If we want to make this change it is best to do it as a separate card.

Reply:
OK. Then let's make the task a separate issue.

Reply:
Task created.